cmd/alpamon/command/ftp.go

```diff
@@ -0,0 +1,32 @@
+package command
+
+import (
+	"github.com/alpacanetworks/alpamon-go/pkg/config"
+	"github.com/alpacanetworks/alpamon-go/pkg/logger"
+	"github.com/alpacanetworks/alpamon-go/pkg/runner"
+	"github.com/spf13/cobra"
+)
+
+var ftpCmd = &cobra.Command{
```

Comment:
`RunE` is used without returning errors. Either add error handling at each step or switch to `Run`.